.ci/install_qemu.sh

```diff
@@ -26,6 +26,9 @@ get_packaged_qemu_commit() {
 	elif [ "$ID" == "centos" ]; then
 		qemu_commit=$(sudo yum --showduplicate list $PACKAGED_QEMU \
 			| awk '/'$PACKAGED_QEMU'/ {print $2}' | cut -d'-' -f1 | cut -d'.' -f4)
+	elif [[ "$ID" =~ ^opensuse.*$ ]]; then
+		qemu_commit=$(sudo zypper info $PACKAGED_QEMU \
+			| grep "Version" | cut -d ':' -f2 | cut -d '-' -f1 | cut -d '.' -f4)
```

Comment:
Suggested to move after the `if..elif` code block the last two piped `cut` commands that are now repeated for all distros. Also, you could use a single `sed` command:
` sed -E "s/.+\+git\.([0-9a-f]+).+/\1/"`